examples/greet-and-react/index.js

```diff
@@ -9,10 +9,26 @@ const SlackStrategy = require('@aoberoi/passport-slack').default.Strategy;
 const http = require('http');
 const express = require('express');
 
+/**
```

Comment:
small nit: could we put this helper function at the bottom of the file (or even import it from another file)? i think having this so close to the top captures too much attention for a beginner who is coming here to learn about how to use the `@slack/events-api` package.